src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -51,6 +51,7 @@
         MEMTABLE_FLUSH_PERIOD_IN_MS,
         MIN_INDEX_INTERVAL,
         SPECULATIVE_RETRY,
+        UPGRADE_CHEAP_QUORUM,
```

Comment:
what would you think about calling this something like `TRANSIENT_WRITE` or `TRANSIENT_WRITE_THRESHOLD` or something similar? I think that would be a more self explanatory name. Not everyone is going to know that cheap quorums are part of transient replication